data/network/src/commonMain/kotlin/social/androiddev/common/network/MastodonApi.kt

```diff
@@ -77,4 +78,11 @@ interface MastodonApi {
      * @return an instance entity
      */
     suspend fun getInstance(domain: String? = null): Result<Instance>
+
+    /**
+     * Fetch home feed for a particular user
+     * @param accessToken representing the user
+     * @return a list of [Status]
+     */
```

Comment:
Can you add a link, similar to the other methods on this interface, to the official doc for this model

Reply:
Also, does the payload contain any other metadata such as "pageSize", "skip", "count" etc, for pagination support?

Reply:
Absolutely, I'll add the rest of the fields we need as well. This was just getting end to end working, definitely in draft so far (forgot to mark as draft)

Reply:
yeah there is pagination support through
```
max_id
String. Return results older than ID.
since_id
String. Return results newer than ID.
min_id
String. Return results immediately newer than ID.
limit
Integer. Maximum number of results to return. Defaults to 20 statuses. Max 40 statuses.
```

Added comment as well